spec/forms/privacy_policy_form_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe Forms::PrivacyPolicyForm, type: :model do
+  describe "Privacy policy URL" do
+    it "is invalid if blank" do
+      privacy_policy_form = described_class.new(privacy_policy_url: "")
+      error_message = I18n.t("activemodel.errors.models.forms/privacy_policy_form.attributes.privacy_policy_url.blank")
+
+      privacy_policy_form.validate(:privacy_policy_url)
+
+      expect(privacy_policy_form.errors.full_messages_for(:privacy_policy_url)).to include(
+        "Privacy policy url #{error_message}",
+      )
+    end
+
+    it "is invalid if URL is not in the correct format" do
+      privacy_policy_form = described_class.new(privacy_policy_url: "laura.mipsum")
+      error_message = I18n.t("activemodel.errors.models.forms/privacy_policy_form.attributes.privacy_policy_url.invalid_url")
+
+      privacy_policy_form.validate(:privacy_policy_url)
+
+      expect(privacy_policy_form.errors.full_messages_for(:privacy_policy_url)).to include(
+        "Privacy policy url #{error_message}",
+      )
+    end
```

Comment:
Could replace this with a single line 

https://github.com/perfectline/validates_url#with-rspec

Reply:
No problem, I can pick this up.

Reply:
[0a2e743](https://github.com/alphagov/forms-admin/pull/108/commits/0a2e7430f43c83c9b5715faad8dc66665df69e94)